### DISCUSSION TITLE
Only getting audio tracks but my video is black when using BRAW Toolbox with FCPX

### DISCUSSION THREAD

**Discussion by yobby27** [latenitefilms/BRAWToolbox#170] [Support]
I am getting the following error each time I drag the green box into my FCPX Library:

The effect ID is invalid. (ref="r4": /fcpxml[1]/event[1]/sync-clip[1]/video[1]/filter-video[1]/@ref)

After I click continue, I only have the XML audio file with no video showing up. Any advice or suggestions would be appreciated.

**Comment by latenitefilms** [comment]
It sounds like the BRAW Toolbox Motion Template hasn't been installed.

Please follow these steps:

https://brawtoolbox.io/troubleshooting/#the-effect-id-is-invalid
